packages/dapp/src/i18n/en.json

```diff
@@ -50,15 +50,16 @@
     "switchToMainnet": "Switch to Mainnet"
   },
   "multisig": {
+    "button": {
+      "cta": "Okay"
+    },
```

Comment:
I meant to add a button as a top-level key so everyone can use it in the future, not only multisig part. Also, let's name the key `okay`, `cta` is a too general term.

Reply:
moving it up